ui/packages/catalog-ui-search/src/main/webapp/component/result-selector/result-selector.less

```diff
@@ -15,15 +15,19 @@
 
   .resultSelector-list {
     overflow: auto;
-    height: ~'calc(100% - 2*@{minimumButtonSize} - @{minimumLineSize})';
+    height: ~'calc(100% - 2.5*@{minimumButtonSize} - @{minimumLineSize})';
```

Comment:
Based on the change to `.resultSelector-menu` this should actually be:
`height: ~'calc(100% - 2*@{minimumButtonSize} - @{minimumLineSize} - 2*@{minimumSpacing})';`